archives/Lemurian_guild.md

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Let's rename this file to something like `guild_tour.md`, otherwise it's confusing the difference between this and `guild.md`.

Reply:
Done